dev-python/shiboken/files/shiboken-5.11-fix-warnings.patch

```diff
@@ -0,0 +1,11 @@
+--- a/libshiboken/pep384impl.cpp
++++ b/libshiboken/pep384impl.cpp
+@@ -130,7 +130,7 @@ check_PyTypeObject_valid(void)
+     long probe_tp_dictoffset = PyLong_AsLong(d);
+     PyObject *probe_tp_mro = PyObject_GetAttrString(obtype, "__mro__");
+     if (false
+-        || probe_tp_name            != check->tp_name
++        || (probe_tp_name - check->tp_name) != 0 // to avoid warning
```

Comment:
what was the warning? does it cause a build failure? if not, I'd rather avoid adding this patch.

Reply:
* QA Notice: Package triggers severe warnings which indicate that it
 *            may exhibit random runtime failures.
 * /var/tmp/portage/dev-python/shiboken-5.11.2/work/pyside-setup-everywhere-src-5.11.2/sources/shiboken2/libshiboken/pep384impl.cpp:133:47: warning: comparison with string literal results in unspecified behavior [-Waddress]

The patch has been taken from qt bugs or forums, can't find this now
5.12 has slightly another solution http://code.qt.io/cgit/pyside/pyside-setup.git/tree/sources/shiboken2/libshiboken/pep384impl.cpp?h=v5.12.0&id=0997b2055007da282cd52087632a28645e3007d1#n133

Reply:
Here is origin: http://code.qt.io/cgit/pyside/pyside-setup.git/commit/?h=5.11.3&id=70d241713ae375124eaabdd928fcdfd034822c6e